UPGRADE.md

```diff
@@ -1,5 +1,18 @@
 # Upgrade to 2.15
 
+## Changing the way how reflection-based mapping drivers report fields, deprecated the "old" mode
```

Comment:
Now that 2.15 has been released, we need to target 2.16.

Reply:
Done